pkg/kubectl/cmd/config/create_authinfo.go

```diff
@@ -90,7 +103,25 @@ var (
 		kubectl config set-credentials cluster-admin --auth-provider=oidc --auth-provider-arg=client-id=foo --auth-provider-arg=client-secret=bar
 
 		# Remove the "client-secret" config value for the OpenID Connect auth provider for the "cluster-admin" entry
-		kubectl config set-credentials cluster-admin --auth-provider=oidc --auth-provider-arg=client-secret-`)
+		kubectl config set-credentials cluster-admin --auth-provider=oidc --auth-provider-arg=client-secret-
+
+		# Enable new exec auth pligin for the "cluster-admin" entry
+		kubectl config set-credentials cluster-admin --exec-command=/path/to/the/executable
+
+		# Specify an exec API version other than the default for the "cluster-admin" entry
+		kubectl config set-credentials cluster-admin --exec-command=/path/to/the/executable --exec-api-version=client.authentication.k8s.io/v1
+
+		# Enable new exec auth pligin for the "cluster-admin" entry with additional args
+		kubectl config set-credentials cluster-admin --exec-command=/path/to/the/executable --exec-arg=arg1 --exec-arg=arg2
+
+		# Remove the "arg-to-remove" argument for the exec plugin for the "cluster-admin" entry
+		kubectl config set-credentials cluster-admin --exec-command=/path/to/the/executable --exec-arg-remove=arg-to-remove
+
+		# Enable new exec auth plugin for the "cluster-admin" entry with additional environment variables
+		kubectl config set-credentials cluster-admin --exec-command=/path/to/the/executable --exec-env=key1=val1 --exec-env=key2=val2
+
+		# Remove the "env-to-remove" environment variable for the exec plugin for the "cluster-admin" entry
```

Comment:
I don't think we need an example for every flag... 1-2 examples showing exec auth usage, and depend on flag help for the remainder

Reply:
Done